spec/tapioca/compilers/dsl/url_helpers_spec.rb

```diff
@@ -233,11 +236,10 @@ class Application < Rails::Application
 
       expected = <<~RBI
         # typed: strong
-        module ActionDispatch
-          class IntegrationTest
-            include GeneratedPathHelpersModule
-            include GeneratedUrlHelpersModule
-          end
+
+        class ActionDispatch::IntegrationTest
+          include GeneratedUrlHelpersModule
+          include GeneratedPathHelpersModule
```

Comment:
Now in the same order we want them to be generated:

![image](https://user-images.githubusercontent.com/583144/125708648-e1ff1b63-89a2-4083-b4d8-c61abe4bc5cf.png)